ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml

```diff
@@ -19,16 +19,16 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-9-release-golang-1.23-openshift-4.19
+    tag: rhel-9-release-golang-1.25-openshift-4.21
 images:
 - additional_architectures:
   - arm64
   dockerfile_path: Dockerfile.ci
   from: base
   inputs:
-    openshift_release_rhel-9-release-golang-1.23-openshift-4.19:
+    openshift_release_rhel-9-release-golang-1.25-openshift-4.21:
       as:
-      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.23-openshift-4.19
+      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.25-openshift-4.21
```

Comment:
@rhmdnd @Vincent056 This should perhaps have been `registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.25-openshift-4.21`, to align with:
https://github.com/ComplianceAsCode/compliance-operator/pull/1026/files#diff-3aaadfe559b0ea24d6dc905d46cee17a924a3f0c047906c270d00254d5258ef2R2